src/components/NewWorkflow/NewWorkflow.js

```diff
@@ -78,16 +78,21 @@ class NewWorkflow extends Component {
                                 <textarea defaultValue={this.state.workflow.description} onChange={(event)=>this.handleChange(event, "inputDescription")}></textarea>
                             </label>  
                             <br/>                   
-                            <button onClick={this.createWorkflow}>Create</button>
+                            <button className="button" onClick={this.createWorkflow}>Create</button>
```

Comment:
this is also not super important, but i think css can grab button types without having to give buttons className. 
so instead of " .button { ... " in css it would just be " button { ... "
i don't mind either way! It might help prevent accidentally missing giving buttons classNames

Reply:
so this is because of my amazing naming conventions. the class of .button has some css atached to that name.... i know creative. but instead of giving the css direct to the <button> html tag i gave it to a classname, so this way not every button auto becomes that style. but i should be more descriptive, i was thinkking to make .buttonLG and .buttonSM or whatever, but not there yet